stacker/config.py

```diff
@@ -1,8 +1,7 @@
+import yaml
```

Comment:
So I tend to break up imports in three sections:

- python native modules
- third party modules
- project modules

That's why this was separate.  I think there's some benefit to doing that - even if you alphabetize inside them.